src/FixedDataTable.js

```diff
@@ -1026,25 +1026,12 @@ var FixedDataTable = createReactClass({
         );
 
         var previousColumnsWidth = 0;
-        for (i = 0; i < scrollableColumnIndex; ++i) {
```

Comment:
If we include the current column, then this isn't really 'previousColumnWidth' is it? And in the (previous) math, selectedColumnWidth is double counted then?